docs/quickstart/prerequisites.mdx

```diff
@@ -2,8 +2,15 @@
 title: "Prerequisites"
 sidebarTitle: "Prerequisites"
 ---
-Before installing Odigos, you need to set up some tools on your computer. These are one-time installations that prepare your computer for running Odigos.
 
+This guide is perfect for you if:
+
+* You're new to tech tools but curious about ways to understand how your computer systems work. Maybe you've heard about "observability" and want to see what it's all about, without getting lost in technical jargon.
+* You're a software developer or work in IT, and while you're comfortable with computers, you're new to Odigos. You're looking for a clear, step-by-step guide that doesn't assume you're already an expert.
+* You're simply interested in Odigos and want to give it a try. This guide will walk you through the process without assuming any prior knowledge.
```

Comment:
I think most of the people that will read our docs do not belong to these categories
should we suggest them to jump to the next step without needing to read all this text?